lib/bugsnag/configuration.rb

```diff
@@ -173,6 +177,13 @@ def should_notify_release_stage?
       @release_stage.nil? || @notify_release_stages.nil? || @notify_release_stages.include?(@release_stage)
     end
 
+    ##
+    # Regex used to mark stacktrace lines as within the project. Lines marked
+    # as "out of project" will only appear in the full trace.
+    def vendor_path
```

Comment:
Add an `@return` comment (or not if you choose to use `attr_accessor` instead and set it to the default in initialize)